src/main/java/org/candlepin/subscriptions/tally/TallySnapshotController.java

```diff
@@ -22,11 +22,9 @@
 
 import io.micrometer.core.annotation.Timed;
 import java.time.OffsetDateTime;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
 import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Please use Spring's StringUtils `hasLength` method.  That class is what we use throughout most of the rest of the codebase

Reply:
@awood Sure I can replace it. I actually used it so, we don't have to trim the whitespace. But, I assume orgId will never be incorrect value " " (empty whitespace)? Correct?

Reply:
You can use `StringUtils.hasText` to check for that.  Here's the relevant source.

```java
	public static boolean hasText(@Nullable String str) {
		return (str != null && !str.isEmpty() && containsText(str));
	}

	private static boolean containsText(CharSequence str) {
		int strLen = str.length();
		for (int i = 0; i < strLen; i++) {
			if (!Character.isWhitespace(str.charAt(i))) {
				return true;
			}
		}
		return false;
	}
```

Reply:
Addressed code review changes.